Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,54 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+    
+# Do you like nature?
+default persistent._mas_pm_likes_nature = None
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_nature",category=['nature', 'you'],prompt="The outdoors",random=True))
+ 
+label monika_nature:
+    m 2esd "Hey, [player]..."
+    $ _history_list.pop()
+    menu:
+        m "Do you like nature{fast}?"
+        "Yes.":
+            $ persistent._mas_pm_likes_nature = True
+            m 2sub "Really? That's wonderful!"
+            m 3wud "You know, I think nature is something we should cherish."
+            m 3eua "Not only is it beautiful, but it also helps humanity out."
+            m 4esc "Insects pollinate our crops, trees give us wood and shade, pets offer us companionship..."
+            m 3esd "Most of all, autotrophs such as plants and algae produce food and oxygen; without them, most life on Earth wouldn't exist."
+            m 1esa "Because of that, I think it's only fair that we give something back to nature since it does so much for us."
+            m 7hub "So, here's Monika's Green Tip of the Day!"
+            m 7eud "Sometimes, people hesitate to go green because they're worried that it's too expensive."
+            m 4esd "That's partially true. Electric vehicles, smart homes, and solar roofs can all cost a fortune."
+            m 3wuo "However, you can make a difference and {i}save{/i} money just by making a few simple choices each day!"
+            m 3eub "Turning off appliances, taking shorter showers, buying a reusable water bottle, commuting by public transport..."
```

Comment:
We could change this to 4eub to mix things up if you'd like.

![1](https://user-images.githubusercontent.com/45746485/93429149-fbcac280-f8c0-11ea-8d2d-187c4176cbdb.png)

Reply:
Sounds good to me.